src/creatures/players/player.hpp

```diff
@@ -1383,7 +1383,7 @@ class Player final : public Creature, public Cylinder, public Bankable {
 	std::string name;
 	std::string guildNick;
 	std::string loyaltyTitle;
-	std::string mapShader;
+	std::string_view mapShader;
```

Comment:
Here, it will remain as std::string. std::string_view is used to avoid copying the string, but in the final context where it needs to be stored or modified, it must be a std::string.